paver.sh

```diff
@@ -552,6 +552,12 @@ detect_os() {
                 check_os_version "SUSE Linux Enterprise Server" 10 \
                     "$os_version_raw" os_version_chevah
                 OS="sles${os_version_chevah}"
+                if [ ${os_version_chevah} -eq 11 ]; then
+                    # Check for SLES 11 with OpenSSL 1.0 (aka Security Module).
+                    if [ -n $(rpm -q openssl1) ]; then
```

Comment:
do we need to execute RPM here? can we just look for a file / path which we know that will exists with OpenSSL 1.0 ?

I don't want to have the paver.sh scrip to heavy and dependent on to many executables